tests/test_main.py

```diff
@@ -113,6 +112,7 @@ def test_deploy_manifest_shinyapps(self):
             httpretty.GET,
             "https://api.shinyapps.io/v1/users/me",
             body=open("tests/testdata/rstudio-responses/get-user.json", "r").read(),
+            adding_headers={"Content-Type": "application/json"},
```

Comment:
For tests to pass, I added this `"Content-Type": "application/json"` header to some of the mock HTTP endpoints. I believe that's the correct behavior to mock the endpoint.